v4l2-request-test.c

```diff
@@ -64,42 +64,51 @@ struct format_description formats[] = {
 
 static void print_help(void)
 {
-	printf("Usage: v4l2-request-test [OPTIONS] [SLICES PATH]\n\n"
+	printf("Usage: v4l2-request-test [OPTIONS]\n\n"
 	       "Options:\n"
-	       " -v [video path]                path for the video node\n"
-	       " -m [media path]                path for the media node\n"
-	       " -d [DRM path]                  path for the DRM node\n"
-	       " -D [DRM driver]                DRM driver to use\n"
-	       " -s [slices filename format]    format for filenames in the slices path\n"
-	       " -f [fps]                       number of frames to display per second\n"
-	       " -P [video preset]              video preset to use\n"
-	       " -i                             enable interactive mode\n"
-	       " -l                             loop preset frames\n"
-	       " -q                             enable quiet mode\n"
-	       " -h                             help\n\n"
-	       "Video presets:\n");
+	       " -v, --device <dev>\n"
+	       "     --video-device <dev>\n"
+	       "                    Use device <dev> as the video device.\n"
+	       " -m, --media-device <dev>\n"
+	       "                    Use device <dev> as the media device.\n"
+	       " -d, --drm-device <dev>\n"
+	       "                    Use device <dev > as DRM device.\n"
+	       " -D, --drm-driver <name>\n"
+	       "                    Use given DRM driver.\n"
+	       " -s, --slices-path <path>\n"
+	       "                    Use <path> to find stored video slices.\n"
+	       " -S, --slices-format <slices format>\n"
+	       "                    Regex/format describing filenames stored in the slices path.\n"
+	       " -f, --fps <fps>\n"
+	       "                    Display given number of frames per seconds.\n"
+	       " -P, --preset-name <name>\n"
+	       "                    Use given preset-name for video decoding.\n"
+	       " -i, --interactive  Enable interactive mode.\n"
+	       " -l, --loop         loop preset frames.\n"
+	       " -q, --quiet        enable quiet mode.\n"
+	       " -h, --help         help\n\n");
 
 	presets_usage();
 }
 
 static void print_summary(struct config *config, struct preset *preset)
 {
 	printf("Config:\n");
-	printf(" Video path: %s\n", config->video_path);
-	printf(" Media path: %s\n", config->media_path);
-	printf(" DRM path: %s\n", config->drm_path);
-	printf(" DRM driver: %s\n", config->drm_driver);
-	printf(" Slices path: %s\n", config->slices_path);
-	printf(" Slices filename format: %s\n", config->slices_filename_format);
-	printf(" FPS: %d\n\n", config->fps);
+	printf(" Video device:  %s\n", config->video_path);
+	printf(" Media device:  %s\n", config->media_path);
+	printf(" DRM device:    %s\n", config->drm_path);
+	printf(" DRM driver:    %s\n", config->drm_driver);
+	printf(" Slices path:   %s\n", config->slices_path);
+	printf(" Slices format: %s\n", config->slices_filename_format);
+	printf(" Frames p/sec:  %d\n\n", config->fps);
```

Comment:
Please keep it called FPS to match the argument name.

Reply:
ok.